examples/cryptorealtime/README.md

```diff
@@ -0,0 +1,159 @@
+# CryptoRealTime
+
+## A Google Cloud Dataflow/Cloud Bigtable Websockets example
+
+The last year has been like a roller coaster for the cryptocurrency market. At the end of 2017, the value of bitcoin (BTC) almost reached $20,000 USD, only to fall below $4,000 USD a few months later. What if there is a pattern in the high volatility of the cryptocurrencies market? If so, can we learn from it and get an edge on future trends? Is there a way to observe all exchanges in real time and visualize it on a single chart? 
+
+In this tutorial we will graph the trades, volume and time delta from trade execution until it reaches our system (an indicator of how close to real time we can get the data).
+
+
+![realtime multi exchange BTC/USD observer](crypto.gif)
+
+[Consider reading the Medium article](https://medium.com/@igalic/bigtable-beam-dataflow-cryptocurrencies-gcp-terraform-java-maven-4e7873811e86)
+
+[Terraform - get this up and running in less than 5 minutes](https://github.com/galic1987/professional-services/blob/master/examples/cryptorealtime/TERRAFORM-README.md)
+
+## Architecture 
+![Cryptorealtime Cloud Architecture overview](https://i.ibb.co/dMc9bMz/Screen-Shot-2019-02-11-at-4-56-29-PM.png)
+
+## Frontend  
+![Cryptorealtime Cloud Fronted overview](https://i.ibb.co/2S28KYq/Screen-Shot-2019-02-12-at-2-53-41-PM.png)
+
+## Costs
+This tutorial uses billable components of GCP, including:
+- Cloud Dataflow
+- Cloud Compute Engine
+- Google Cloud Storage
```

Comment:
Drop "Google", just "Cloud Storage" in this context.

Reply:
8af3520